validation/validator.go

```diff
@@ -9,8 +9,15 @@ import (
 	"github.com/nyaruka/phonenumbers"
 )
 
+// Validable interface
+type Validable interface {
```

Comment:
don't know if this word exists in english. Seems like a franglais word :)

Reply:
Changing to Validatable...